tests/test_client.py

```diff
@@ -301,29 +302,36 @@ def test_bulk_get_latest_with_skip_null(index_client):
     assert {doc.did for doc in docs_null_included} == null_dids
 
 
-def test_bulk_get_latest_with_skip_deleted(index_client):
-    """ Test bulk_get_latest() with skip_deleted parameters """
+def test_bulk_get_latest_with_exclude_deleted(index_client):
+    """
+    Tests bulk_get_latest() with exclude_deleted parameter
```

Comment:
This is kinda a restatement of the name of the test function itself. If we are adding test comments we should prob add value in some way beyond that. Maybe add more detail about how the test does this?